dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1140,6 +1141,19 @@ private void wesServiceInfo(WorkflowExecutionServiceApi clientWorkflowExecutionS
         }
     }
 
+    /**
+     * This will attempt to retrieve information regarding the WES server
+     * @param clientWorkflowExecutionServiceApi The API client
+     */
+    private void wesListRuns(int pageSize, String pageToken, WorkflowExecutionServiceApi clientWorkflowExecutionServiceApi) {
+        try {
+            RunListResponse response = clientWorkflowExecutionServiceApi.listRuns((long)pageSize, pageToken);
+            out("WES Run List: " + response.toString());
```

Comment:
Feature creep ideas:

1. Read the next page token from the response. If it's null say there are no more runs. If it's not null, print out the CLI command to get the next batch, e.g., `dockstore wes list --count <value of pageSize parameter> --page-token <value of next page token>`
2. Don't even surface the page-token. Just have `dockstore wes list --count <number of records>`, then the CLI does the pagination behind the scenes necessary to fetch `<number of records>`.

Optional and/or can be spun out into a separate issue.

Reply:
Good suggestions, I think removing `--page-token` entirely will improve usability the most. Created a followup issue for this: https://github.com/dockstore/dockstore/issues/4679